staging/src/k8s.io/client-go/dynamic/client.go

```diff
@@ -205,11 +220,26 @@ func (rc *ResourceClient) DeleteCollection(deleteOptions *metav1.DeleteOptions,
 // Create creates the provided resource.
 func (rc *ResourceClient) Create(obj *unstructured.Unstructured) (*unstructured.Unstructured, error) {
 	result := new(unstructured.Unstructured)
-	err := rc.cl.Post().
+	resourceName, subresourceName := rc.parseResourceSubresourceName()
+	req := rc.cl.Post().
 		NamespaceIfScoped(rc.ns, rc.resource.Namespaced).
-		Resource(rc.resource.Name).
-		Body(obj).
-		Do().
+		Resource(resourceName).
+		Body(obj)
+	if len(subresourceName) > 0 {
+		// If the provided resource is a subresource, the POST request should contain
+		// object name. Examples of subresources that support Create operation:
+		//	core/v1/pods/binding
+		//	core/v1/pods/eviction
+		//	extensions/v1beta1/deployments/rollback
+		//	apps/v1beta1/deployments/rollback
+		// NOTE: Currently our system assumes every subresource object has the same
```

Comment:
The `Update()` function has the same assumption. Could you copy-paste this NOTE to there as well?